### PR TITLE
Collect attribute information on eval, if available

### PR DESCRIPTION
**Pull request by wesrupert**
Missed a code path for getting variable information in the last PR. Also pull attribute data from Eval if it exists.

### CODE REVIEW

**Comment by wesrupert** [comment]
@gregg-miskelly @caslan @chuckries @rajkumar42

**Comment by gregg-miskelly** [comment]
:shipit:

**Comment by chuckries** [comment]
:+1:

File: src/MIDebugEngine/Engine.Impl/Variables.cs [hunk 1]
```diff
@@ -466,6 +466,15 @@ public string EvalDependentExpression(string expr)
                     {
                         DisplayHint = results.FindString("displayhint");
                     }
+                    if (results.Contains("attributes"))
+                    {
+                        _isReadonly = false;
```
Comment (rajkumar42) [inline src/MIDebugEngine/Engine.Impl/Variables.cs]: this is redundant. _isReadOnly = false, can be moved to the constructor or class initializer. 
I see a similar pattern in line 238

**Comment by rajkumar42** [comment]
:shipit:

**Pull request merged**